setup.py

```diff
@@ -1,8 +1,13 @@
+from os import sys, path
+
 from skbuild import setup
 
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from itkVersion import VERSION
```

Comment:
I think it is a good idea to keep `itkVersion.py` in the `ITKPythonPackage` repository since it is then scoped to the Python package.

We could add it to `py_modules` if there is a need, but folks will hopefully use the traditional `itk.__version__` anyway.